src/config_objects.c

```diff
@@ -325,6 +411,33 @@ TbBool parse_objects_object_blocks(char *buf, long len, const char *config_textn
             skip_conf_to_next_line(buf,&pos,len);
         }
 #undef COMMAND_TEXT
+        if (anim_path[0] != 0)
+        {
+            if (sprite_max_size <= 0)
+            {
+                CONFWRNLOG("Custom sprite without max_size [%s] block of %s file.",
+                           block_buf, config_textname);
+            }
+            else
+            {
+                short anim_idx = add_custom_sprite(prepare_file_path(FGrp_FxData, anim_path),
```

Comment:
Simply putting this in the 'fxdata' folder is a bit messy. Would be a lot nicer to be able to configure an 'OBJECTS_LOCATION' in the campaign/mappack configs.

Reply:
Could you please explain it more. Is it some defined value or I should add it?

Reply:
This should be added. If you agree with it, I can help with that bit too.